types/azdata/tslint.json

```diff
@@ -10,6 +10,7 @@
         "no-single-declare-module": false,
         "no-unnecessary-generics": false,
         "npm-naming": false,
-        "strict-export-declare-modifiers": false
+        "strict-export-declare-modifiers": false,
+        "unified-signatures": false
```

Comment:
Seems like unified-signatures should add an exception when any of the signatures is deprecated.

Reply:
Agreed, thank you for submitting that!